packages/http/src/router/__tests__/matchPath.spec.ts

```diff
@@ -164,8 +164,15 @@ describe('matchPath()', () => {
     assertRight(matchPath(requestPath, operationPath), e => expect(e).toEqual(MatchType.NOMATCH));
   });
 
+  test('none path should match templated operation with more path fragments', () => {
+    const requestPath = '/a/b/c';
```

Comment:
This specific test case has been extracted from the one above when I caught it failing during a CI build. 

Most of the other dynamically generated test cases were comparing against **partially** templated paths (eg `/a/b/c` vs `/{d}/e/{f}/g`).